packages/fx-core/src/core/FxCore.ts

```diff
@@ -1143,16 +1150,69 @@ export class FxCore {
     const context = createContextV3();
     return await coordinator.publishInDeveloperPortal(context, inputs as InputsWithProjectPath);
   }
+
+  async injectCreateAPIKeyAction(
+    ymlPath: string,
+    authName: string,
+    specRelativePath: string
+  ): Promise<void> {
+    const ymlContent = await fs.readFile(ymlPath, "utf-8");
+
+    const document = parseDocument(ymlContent);
+    const provisionNode = document.get("provision") as any;
+
+    if (provisionNode) {
+      const hasApiKeyAction = provisionNode.items.some(
+        (item: any) =>
+          item.get("uses") === "apiKey/create" && item.get("with")?.get("name") === authName
```

Comment:
do we have central place for action names?

Reply:
No, I don't find central place of these names, each action hold its own action name.